packages/mdc-select/README.md

```diff
@@ -58,7 +58,19 @@ npm install @material/select
 </div>
 ```
 
-Then with JS
+### Styles
+
+```scss
+@import "@material/select/mdc-select";
+
+// Invoke mixins in the context of a custom class applied to the component's root element:
```

Comment:
Is this necessary to get the default select? If not, it doesn't need to go under Basic Usage.

Reply:
I had done this as a replacement for the remark about including the mixins within a `.foo-select` custom class selector in the Sass mixins section, but I can put that back.